Travel Scheduler/View Controllers/Place.m

```diff
@@ -101,4 +118,133 @@ - (void)getListOfPlacesCloseToPlaceWithName:(NSString *)centerPlaceName withComp
     }];
 }
 
+#pragma mark - Methods to set type
+-(void)setPlaceSpecificType {
+    if([self.types containsObject:@"restaurant"]) {
+        self.specificType = @"restaurant";
+    }
+    else if([self.types containsObject:@"lodging"]) {
+        self.specificType = @"hotel";
+    }
+    else {
+        self.specificType = @"attraction";
+    }
+}
+
+#pragma mark - methods to make the dictionary of opening times
+- (void)makeDictionaryOfOpeningTimes{
+    for(int dayIndexInt = 0; dayIndexInt <= 6; ++dayIndexInt){
+        NSNumber *dayIndexNSNumber = [[NSNumber alloc] initWithInt:dayIndexInt];
+        [self formatTimeForDay:dayIndexNSNumber];
+    }
+}
+
+-(void)formatTimeForDay:(NSNumber *)day {
+    int dayInt = [day intValue];
```

Comment:
It seems like you are only using dayInt in this function and not day. If you don't need the specific NSNumber reference, you could have removed line 137 above and just passed in dayIndexInt into formatTimeForDay so it would look like:
- (void)formateTimeForDay:(int)dayInt {... 
This way, you don't need to convert it into an NSNumber in line 137 and convert it back.